keyboards/cue2keys/cue2keys.c

```diff
@@ -0,0 +1,406 @@
+        dprintf("GP9 pressed\n");
+        display_mode = (display_mode + 1) % DisplayMode_MAX;
+#ifdef OLED_ENABLE
+        oled_clear();
+#endif
+        return false;
+    }
+
+    // process next oled page
+    if (record->event.pressed) {
+        switch (keycode) {
+            case NEXT_OLED_PAGE:
+                display_mode = (display_mode + 1) % DisplayMode_MAX;
+#ifdef OLED_ENABLE
+                oled_clear();
+#endif
+                return false;
+        }
+    }
+
+    // process angle change
+    if (record->event.pressed) {
+        {
+            const uint16_t value = 15;
+            switch (keycode) {
+                case ROT_R30_1:
+                    update_half_angle(0, value);
+                    return false;
+                case ROT_L30_1:
+                    update_half_angle(0, -value);
+                    return false;
+                case ROT_R30_2:
+                    update_half_angle(1, value);
+                    return false;
+                case ROT_L30_2:
+                    update_half_angle(1, -value);
+                    return false;
+            }
+        }
+        {
+            const uint16_t value = 1;
+            switch (keycode) {
+                case ROT_R2_1:
+                    update_half_angle(0, value);
+                    return false;
+                case ROT_L2_1:
+                    update_half_angle(0, -value);
+                    return false;
+                case ROT_R2_2:
+                    update_half_angle(1, value);
+                    return false;
+                case ROT_L2_2:
+                    update_half_angle(1, -value);
+                    return false;
+            }
+        }
+    }
+
+    // process mouse layer ms change
+    if (record->event.pressed) {
+        switch (keycode) {
+            case MOUSE_LAYER_MS_ADD_100MS:
+                user_config.mouse_layer_ms += 1;
+                if (user_config.mouse_layer_ms > 40) user_config.mouse_layer_ms = 40;
+                eeconfig_update_kb(user_config.raw);
+                return false;
+            case MOUSE_LAYER_MS_MINUS_100MS:
+                user_config.mouse_layer_ms -= 1;
+                // overflow
+                if (user_config.mouse_layer_ms > 40) user_config.mouse_layer_ms = 0;
+                eeconfig_update_kb(user_config.raw);
+                return false;
+        }
+    }
+
+        // process pointer move speed change
+        if (record->event.pressed) {
+            switch (keycode) {
+                case POINTER_SPEED_MAG_CHANGE:
+                    user_config.pointer_speed_magnification += 1;
+                    if (user_config.pointer_speed_magnification > 3) user_config.pointer_speed_magnification = 0;
+                    eeconfig_update_kb(user_config.raw);
+                    return false;
+            }
+        }
+
+    return process_record_user(keycode, record);
+}
+
+#ifdef OLED_ENABLE
+bool oled_task_kb(void) {
```

Comment:
The oled_task_user should be called like this:
```suggestion
bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
```

Otherwise, there is no way for the user to override the functionality without editing the kb files.